src/data/markdown/docs/02 javascript api/09 k6-net-grpc/20 Client/10-Client-load-importpaths-protoFiles.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Client.load(importPaths, ...protoFiles)"
+---
+
+Loads and parses the protocol buffer descriptors so they are available to the client to marshal/unmarshal the correct request and repsonse data structures for the RPC schema.
+
+Must be called within the [`init` phase](/using-k6/test-life-cycle).
+
+| Parameter | Type | Description |
+|-----------|------|-------------|
+| importPaths | Array&lt;string&gt; \| `null` | The paths used to search for dependencies that are referenced in import statements in proto source files. If no import paths are provided then "." (current directory) is assumed to be the only import path. |
```

Comment:
We should mention if absolute paths are supported, any compatibility issues on Windows, etc.

Reply:
They are technically supported - you can use absolute paths on windows and on linux. The problems occur when you use absolute paths on Windows with `k6 cloud`, since we then execute the archives on linux machines. It's technically a k6 bug, so we are going to eventually fix it, it's just tricky to do so.

Reply:
I am more for ... not allowing absolute paths ... but regardless I think the wording here is fine